libs/core/src/components/pds-link/pds-link.tsx

```diff
@@ -54,7 +54,12 @@ export class PdsLink {
     );
 
     return (
-      <a id={this.componentId} href={this.href} class={`pds-link pds-link--${this.variant} pds-link--${this.fontSize} `} target={this.external ? '_blank' : undefined}>
+      <a
+        id={this.componentId}
+        href={this.href}
+        class={`pds-link ${this.variant ? `pds-link--${this.variant}` : ''} ${this.fontSize ? `pds-link--${this.fontSize}` : ''}`}
```

Comment:
Can we extract the logic out of here to function like we do the other Components?

e.g 
```
const classNames = () => {
  const classes = ['pds-link']
  
  if ( this.variant) {
    classes.push(`pds-link--${this.variant}`);
  }
  
  if (this.fontSize) {
    classes.push(`pds-link--${this.fontSize}`);
  }
}
```

Reply:
Done. Great catch!